app/views/menu.ejs

```diff
@@ -34,15 +31,40 @@
             <br>
             <p id="menuInfo"></p>
         </div>-->
+
+        <!-- Sample filter (left) *TEMPORARY* -->
+        <div class="filter">
```

Comment:
These don't need to be <a> tags, since they won't be links in the end of things.  They won't navigate us to a different part of the page, instead they will just filter things out, so these can just be divs.